quantum/quantum.c

```diff
@@ -627,14 +634,21 @@ bool process_record_quantum(keyrecord_t *record) {
       }
       else {
         #ifdef DISABLE_SPACE_CADET_ROLLOVER
-          if (get_mods() & MOD_BIT(KC_RSFT)) {
+          if (get_mods() & MOD_BIT(RSPC_MOD)) {
             shift_interrupted[0] = true;
             shift_interrupted[1] = true;
           }
         #endif
         if (!shift_interrupted[0] && timer_elapsed(scs_timer[0]) < TAPPING_TERM) {
+          unregister_mods(MOD_BIT(KC_LSFT));
```

Comment:
Should we avoid to unregister the mod if KC_SLFT == LSPO_MOD by adding a check or is this unimportant?

Reply:
Yes it would make sens, it will try to make it work.